drape/gl_extensions_list.cpp

```diff
@@ -37,6 +37,24 @@ void GLExtensionsList::Init(dp::ApiVersion apiVersion)
     SetExtension(VertexArrayObject, true);
     SetExtension(UintIndices, true);
   }
+#elif defined(OMIM_OS_LINUX)
+  SetExtension(MapBuffer, true);
+  SetExtension(UintIndices, true);
+  if (apiVersion == dp::ApiVersion::OpenGLES2)
+  {
+#if defined(OMIM_OS_LINUX_HAS_EGL)
```

Comment:
"Linux Wayland" is a _mobile_ OS? E.g. if we are checking for  OGL ES here, should we enable OMIM_OS_MOBILE in this case? If yes, should we move this checks (ES) above under the OMIM_OS_MOBILE?

Reply:
But first of all should make this combination (Linux + ES2) works and then prettify defines.

Reply:
+1 to vng

We should choose a proper way for Linux versions. The easiest is probably this one:
Build a separate desktop version with OpenGL, and a mobile version with OpenGL ES.

The more complex way is to build one version that can detect the OpenGL version in runtime and switch appropriately.

Reply:
Or, alternatively, should we rewrite/remove OMIM_OS_MOBILE to dynamically initialize GLES if it's present, and fall back to GL if it's not?